pkg/controller/migmigration/task.go

```diff
@@ -1362,6 +1379,66 @@ func (t *Task) hasImageStreams() (bool, error) {
 	return false, nil
 }
 
+// Get whether the associated plan has relevant artifacts using the analytics resource.
+func (t *Task) hasRelevantArtifacts() (bool, string, error) {
```

Comment:
Could we change occurrences of the term `Artifacts` to `Resources`? This would be more in line with our terminology throughout the codebase. I don't believe artifacts has an official meaning within the k8s ecosystem.

Reply:
Yes. I will make this change.